Web/JQuerry-1/Assignment3/JQuerySelectors.js

```diff
@@ -0,0 +1,53 @@
+//Selects all span with class name: “message”
+$(document).ready(function() {
+	$("span[class='message']").css("background-color","#009ACD");
+	});
+
+//Selects first div in a document that has class: “box”	
+$(document).ready(function() {
```

Comment:
ACTION: No need to add $(document).ready(function() again and again it is for once dom is ready. so write it only once.